ddp-workspace/projects/study-builder-ui/src/app/components/text-question-editor/text-question-editor.component.ts

```diff
@@ -52,15 +57,27 @@ export class TextQuestionEditorComponent implements OnInit, OnDestroy {
         this.sub.unsubscribe();
     }
 
+    changeValidators(validationRules) {
```

Comment:
The eslint on my IDE is is still messed up, but I am hoping that we are still enforcing having to include argument types and return types.

Reply:
Fixed, thanks.